docs/tutorials/Scikit-learn integration.ipynb

```diff
@@ -0,0 +1,312 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Author: Pierre Ablin\n",
+    "\n",
+    "# Integrating mvlearn with scikit-learn\n",
+    "\n",
+    "`mvlearn` mimics most of `scikit-learn` API, and integrates seamlessly with it. In scikit-learn, a dataset is represented as a 2d array `X` of shape `(n_samples, n_features)`. In `mvlearn`, datasets `Xs` are lists of views, which are themselves 2d arrays of shape `(n_samples, n_features_i)`. The number of features does not have to be constant:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n_samples = 100\n",
+    "n_features1 = 20\n",
+    "n_features2 = 30\n",
+    "\n",
+    "X1 = np.random.randn(n_samples, n_features1)\n",
+    "X2 = np.random.randn(n_samples, n_features2)\n",
+    "Xs = [X1, X2]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, `Xs` is a multiview dataset, containing two views. `mvlearn` works with these objects."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ViewTransformer\n",
+    "\n",
+    "`mvlearn.preprocessing.ViewTransformer` is a handy tool to apply the same `sklearn` transformer to each view of the multiview dataset. For instance, it is simple to apply PCA to each view. In the following, we reduce the dimension of each view to 3:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2\n",
+      "[(100, 3), (100, 3)]\n"
+     ]
+    }
+   ],
+   "source": [
+    "from mvlearn.preprocessing import ViewTransformer\n",
+    "from sklearn.decomposition import PCA\n",
+    "\n",
+    "pca = PCA(n_components=3)\n",
+    "mvpca = ViewTransformer(pca)\n",
+    "X_transformed = mvpca.fit_transform(Xs)\n",
```

Comment:
In the ViewTransformer docs, I think the Return is given as Xs_transformed, rather than X_transformed, so maybe keep this differentiation?